config/diff.go

```diff
@@ -16,13 +16,14 @@ import (
 // where left is usually old and right is new. So the diff is the
 // changes from left to right.
 type Diff struct {
-	Left, Right    *Config
-	Added          *Config
-	Modified       *ModifiedConfigDiff
-	Removed        *Config
-	ResourcesEqual bool
-	NetworkEqual   bool
-	PrettyDiff     string
+	Left, Right          *Config
+	Added                *Config
+	Modified             *ModifiedConfigDiff
+	Removed              *Config
+	ResourcesEqual       bool
+	NetworkEqual         bool
+	PrettyDiff           string
+	OnlyModifiedRevision *Config
```

Comment:
now that we don't store revisions in configs anymore, is this still necessary? otherwise would want to change the name of the field to `UnmodifiedResources` or something more specific

Reply:
yeah we still this field [to apply revision updates](https://github.com/viamrobotics/rdk/pull/4206/files#diff-0b975da6011656cd3f2caf0435023573bdb5ac7c6a425e7917dd5cc9d6be63e4R1244-R1250) - I'll update the name.

Reply:
done https://github.com/viamrobotics/rdk/pull/4206/commits/c4c859dcc729e7be86d9a5bf50ffb394b77187d9 - this field just holds `resource.Config`s now since the rest of the config is not important.